packages/system/kairos-agent/build.yaml

```diff
@@ -1,7 +1,14 @@
+{{if eq .Values.category "fips" }}
+requires:
+  - name: "toolchain-go-ubuntu"
+    category: "fips"
+    version: ">=0"
+  {{else}}
```

Comment:
this is nitpicking but could you have the "else" and "end" match the indentation of the "if"?

Reply:
crap. I swear its the editor auto changing this on save